test/utils/image/manifest.go

```diff
@@ -60,11 +60,21 @@ func (i *Config) SetName(name string) {
 	i.name = name
 }
 
+// GetName returns the image name from a Config struct
+func (i *Config) GetName() string {
+	return i.name
+}
+
```

Comment:
This makes me wonder why Config has lowercase fields if we're just adding getters/setters.  This doesn't feel very go-like